tests/dummy/app/templates/docs/migrations.md

```diff
@@ -0,0 +1,45 @@
+# Migrations
+
+### v0.10 -> v0.11
+
+1. `@ember/legacy-built-in-components` no longer required for ember engines.
+    It can be safely removed from your app if it's the only reason it was added.
+
+    ```diff
+    # package.json
+    {
+        "dependencies": {
+    -       "@ember/legacy-built-in-components": "^0.5.0",
+            ...
+        }
+    }
+    ```
+
+2. Add [ember-engines-router-service](https://github.com/villander/ember-engines-router-service) to `dependencies`
```

Comment:
I don’t think so. It should be mandatory given it’s part of the engine RFC. We should make router service for engines accessible as we have for normal ember apps

Reply:
I know it wasn't the intention to complete the RFC, but it ended up completing the RFC. That being said, there's a long wait for this feature/support. So that's not something up to us, but it was something promised since 2016. So I agree with @SergeAstapov here and IMO it must be merged. The second step would be make `ember-engines-router-service` part of the ember-engines addon to complete the RFC